src/cil.ml

```diff
@@ -1428,14 +1404,8 @@ let hasAttribute s al =
 
 
 type attributeClass =
-    AttrName of bool
-        (* Attribute of a name. If argument is true and we are on MSVC then
-         * the attribute is printed using __declspec as part of the storage
-         * specifier  *)
-  | AttrFunType of bool
-        (* Attribute of a function type. If argument is true and we are on
-         * MSVC then the attribute is printed just before the function name *)
-
+    AttrName  (* Attribute of a name. *)
+  | AttrFunType  (* Attribute of a function type. *)
```

Comment:
Also here you could remove all the now unnecessary parenthesis around whereever these constructors are used.